review-notifications/src/mainBg.js

```diff
@@ -53,7 +50,12 @@ chrome.runtime.onMessage.addListener(async request => {
   }
 });
 
+chrome.runtime.onRestartRequired.addListener(onStart());
```

Comment:
are you sure this is correct? As it stands you are invoking the `onStart()` function and passing whatever it returns instead of passing the function itself. Shouldn't this be `onStart`?